tests/integration/tables/process_open_pipes.cpp

```diff
@@ -0,0 +1,233 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+// Sanity check integration test for process_open_pipes
+// Spec file: specs/posix/process_open_pipes.table
+
+#include <osquery/logger.h>
+#include <osquery/tests/integration/tables/helper.h>
+
+#include <fcntl.h>
+#include <signal.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+namespace osquery {
+namespace table_tests {
+
+class ProcessOpenPipesTest : public testing::Test {
+  std::string pipe_path;
```

Comment:
Final nitpick, do you mind updating class member variables to use a trailing `_` so this becomes `pipe_path_` to enhance readability?